### DISCUSSION TITLE
Server display problem

### DISCUSSION THREAD

**Discussion by NexoFr** [Ch4r0ne/ARK-Ascended-Server-Manager#1] [General]
Anglais

We don't see the server on ARK ASCENDED. Do you have a solution ?

**Comment by Ch4r0ne** [comment]
You need to port forward the Ports 7777 TCP/UDP and your Port = 27025 TCP/UDP

Unfortunately, I also have this problem, but no matter how you host the server, it is only listed for me if I install Steam and the app on the server to be hosted. Only then will my server be listed. But from then on everything works and everyone can find him. Strange, possibly a bug from the Ark developers.

**Reply by Kaffiend** [reply]
Is this specifically a public listing issue? If i manually favorite the server in steam and then go into ARK to join it, or use parameters it should still work fine correct?

**Reply by Ch4r0ne** [reply]
I had an issue with both local and public listing, but now I can host without any problems using one of my servers. However, I don't know the reason for this. Later, I added a second server, reinstalled it with the same parameters, but it doesn't appear in the game. 

The last time I checked, the server couldn't be searched through Steam; I believe this feature is yet to be implemented by the ARK team. Having this option would make finding the server much easier.

## Port forward Ports Firewall
- Port = 27015 - UDP 
- QueryPort = 27016 - TCP

Client Firewall Rule:
```
New-NetFirewallRule -DisplayName "ASA Server" -Direction Inbound -LocalPort 27015 -Protocol UDP -Action Allow
New-NetFirewallRule -DisplayName "ASA Server" -Direction Inbound -LocalPort 27016 -Protocol UDP -Action Allow
New-NetFirewallRule -DisplayName "ASA Server" -Direction Outbound -LocalPort 27015 -Protocol TCP -Action Allow
New-NetFirewallRule -DisplayName "ASA Server" -Direction Outbound -LocalPort 27016 -Protocol UDP -Action Allow
```

**Reply by Fr1tzl1** [reply]
Hi
You need to install the EPIC root certificate, because the in-game server list in ASA works via the EPIC API

1. ARK: SA Server Stop
2. Install certificate as described here (https://ark.wiki.gg/wiki/Dedicated_server_setup#Windows_Server_editions_and_crossplay_with_Epic_players)
3. ARK: SA Server Start then it appears in the server list in game

**Reply by Ch4r0ne** [reply]
Hello @Fr1tzl1, you are the best, this has solved my problem safely and reliably. I will include your solution in the install script, thank you very much =)

**Reply by Fr1tzl1** [reply]
Is it possible to contact you via discord, I have some functions that could be added, e.g. scheduled update checks with automatic updates when there is a server update and the players on the server are then informed via rcon with a countdown for server update/restart + worldsave befor shutdown/restart. The same with restart or additional start parameters

**Reply by Ch4r0ne** [reply]
@Fr1tzl1 Thank you for your contribution; you certainly deserve a spot in the release notes for your efforts. https://github.com/Ch4r0ne/ARK-Ascended-Server-Manager/releases/tag/1.0.1

I am active in the ASA Self Server Discord channel https://discord.gg/3eYPmAyt
